code/API_definitions/number_verification.yaml

```diff
@@ -3,16 +3,16 @@ info:
   title: Number Verification
   description: |-
     Service Enabling Network Function API to verify that the provided **mobile phone number** is the one used in the device. It verifies that the user is using a device with the same *mobile phone number* as it is declared.
-    It also make it possible for a Service provider to verify the number itself by returning the phone number retrieved from the authenticated user's access token.
+    It also make it possible for a Service provider to verify the number itself by returning the phone number associated to the authenticated user's access token.
     
-    In this API **phone number** term refers to the mobile phone number (MSISDN)
+    In this API **phone number** term refers to the mobile phone number
 
     # API Functionality 
-    It enables a Service Provider (SP) to verify the phone number (MSISDN) of the mobile device being used to access their service where the mobile device is accessing the *service provider* over a mobile network (WiFi connections are out of this API scope) either by getting the comparison result or receiving the phone number of the device that it is used, so they can verify it themselves.
+    It enables a Service Provider (SP) to verify the phone number of the mobile device being used to access their service where the mobile device is accessing the *service provider* over a mobile network (WiFi connections are out of this API scope) either by getting the comparison result or receiving the phone number of the device that it is used, so they can verify it themselves.
 
     # Resources and Operations overview
     This API currently provides two endpoints where both require a **3-legged token** and authentication via **mobile network** (excluding for example by SMS/OTP or user/password as an authentication method):
-    - The first one checks if the user phone number matches the MSISDN associated with the mobile device. It can receive either a hashed phone number or a clear phone number as input and it compares the received input with the authenticated user's MSISDN retrieved from the access token in order to respond **true/false**.
+    - The first one checks if the user mobile phone number matches the phone number associated with the mobile device. It can receive either a hashed phone number or a clear phone number as input and it compares the received input with the authenticated user's phone number retrieved from the access token in order to respond **true/false**.
```

Comment:
phone number retrieved from the access token --> phone number *associated to* the access token

Reply:
Edited